src/components/studies/StudyBuilder.tsx

```diff
@@ -461,6 +461,7 @@ const StudyBuilder: FunctionComponent<StudyBuilderProps> = ({
                         onSave={(updatedStudy: Study) => {
                           saveStudy(builderInfo.study)
                         }}
+                        sideDrawerIsOpen={open}
```

Comment:
what is this needed for

Reply:
Forgot to take this out after I switched how I was displaying the section indicators. I just removed it.